File: Retos/Reto #40 - TABLA DE MULTIPLICAR [Fácil]/python/George241915.py
'''
Crea un programa que sea capaz de solicitarte un número y se
 * encargue de imprimir su tabla de multiplicar entre el 1 y el 10.
 * - Debe visualizarse qué operación se realiza y su resultado.
 *   Ej: 1 x 1 = 1
 *       1 x 2 = 2
 *       1 x 3 = 3
 *       ... 
'''

def tabla_multiplicar(numero):
    for i in range(1, 11):
        total = numero * i
        print(f"{numero} x {i} = {total}")

numero = int(input("Ingrese un numero: "))
print("Tabla de multiplicar del numero:", numero)
tabla_multiplicar(numero)

File: Retos/Reto #40 - TABLA DE MULTIPLICAR [Fácil]/java/George241915.java
import java.util.Scanner;

public class george24
{
    public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
        int num = 0;
        System.out.println("Ingrese el número: ");
        num = sc.nextInt();
        sc.close();
        System.out.println("Tabla de multiplicar del " + num + ": ");
        tabla_multip(num);

	}

    public static void tabla_multip(int num){
        for (int i= 1; i <=10; i++){
            int total = num *i;
            System.out.println(num + " x " + i + " = " + total );
        }
    }
}